### PR TITLE
Contact backend

### PR DESCRIPTION
**Pull request by MihneaDinuGL**
This is the backend for the contacts carousel and it is linked to the attachment component

### CODE REVIEW

**Comment by MihneaDinuGL** [comment]
> This appears to contain a lot of Stanislav's code which has not finalized?

It does, I will merge dev into my branch once the attachments component is finalised and solve any conflicts

**Comment by MihneaDinuGL** [comment]
> If this the contacts backend, why are there a load of case studies changes in the diffs?

I'm not sure, perhaps it happened with one of the merges? I did not make any changes to the case studies backend

File: src/main/java/com/globallogic/amcr/controller/contacts/ContactsController.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+    /**
+     * @param id the ID of the contacts page object to be retrieved
+     * @return returns a response entity either 200 or 500
+     */
+    @GetMapping(value = "/{id}", produces = "application/json")
+    public ResponseEntity<Contacts> get(@PathVariable UUID id) {
+        LOG.debug("Controller requesting contacts page data with ID {}", id);
+        Contacts createdContact = contactsService.get(id);
+        createdContact.setImageLink(Utils.generateUri("/attachment/{id}", createdContact.getImageId()).toString());
```
[the remark this answers is not in the document]
Comment (MihneaDinuGL) [reply src/main/java/com/globallogic/amcr/controller/contacts/ContactsController.java:55]: The Utils change doesn't seem to be on dev yet, and if I were to split the class myself I would need to update other classes that are not related to my component. Should I wait for the case-studies branch with the new classes split from Utils to be merged into dev?

**Review by S1sO0 — APPROVED** [review]

File: src/main/java/com/globallogic/amcr/controller/contacts/ContactsController.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (MihneaDinuGL) [reply src/main/java/com/globallogic/amcr/controller/contacts/ContactsController.java]: This has been updated

**Pull request merged**